script.js

```diff
@@ -10,3 +10,225 @@ document.querySelectorAll('.mobile-link').forEach((n) => n.addEventListener('cli
+    popTechnologies2: ['Codekit', 'GitHub', 'JavScript', 'Bootstrap', 'Terminal', 'Codepen'],
+    btn1: 'See Live',
+    btn2: 'See Source',
+    linkToLiveSource: '#',
+    linkToSource: '#',
+  },
+  {
+    id: 2,
+    title: 'Multi-Post Stories Gain+Glory',
+    technologies: ['Ruby on rails', 'css', 'Javascript', 'html'],
+    popTechnologies1: ['Ruby on rails', 'css', 'Javascript'],
+    image: '/portfolio-Img/mobile-popup.png',
+    popTitle: 'Keeping track of hundreds of components',
+    popTechnologies2: ['Codekit', 'GitHub', 'JavScript', 'Bootstrap', 'Terminal', 'Codepen'],
+    btn1: 'See Live',
+    btn2: 'See Source',
+    linkToLiveSource: '#',
+    linkToSource: '#',
+  },
+  {
+    id: 3,
+    title: 'Multi-Post Stories Gain+Glory',
+    technologies: ['Ruby on rails', 'css', 'Javascript', 'html'],
+    popTechnologies1: ['Ruby on rails', 'css', 'Javascript'],
+    image: '/portfolio-Img/mobile-popup.png',
+    popTitle: 'Keeping track of hundreds of components',
+    popTechnologies2: ['Codekit', 'GitHub', 'JavScript', 'Bootstrap', 'Terminal', 'Codepen'],
+    btn1: 'See Live',
+    btn2: 'See Source',
+    linkToLiveSource: '#',
+    linkToSource: '#',
+  },
+  {
+    id: 4,
+    title: 'Multi-Post Stories Gain+Glory',
+    technologies: ['Ruby on rails', 'css', 'Javascript', 'html'],
+    popTechnologies1: ['Ruby on rails', 'css', 'Javascript'],
+    image: '/portfolio-Img/mobile-popup.png',
+    popTitle: 'Keeping track of hundreds of components',
+    popTechnologies2: ['Codekit', 'GitHub', 'JavScript', 'Bootstrap', 'Terminal', 'Codepen'],
+    btn1: 'See Live',
+    btn2: 'See Source',
+    linkToLiveSource: '#',
+    linkToSource: '#',
+  },
+  {
+    id: 5,
+    title: 'Multi-Post Stories Gain+Glory',
+    technologies: ['Ruby on rails', 'css', 'Javascript', 'html'],
+    popTechnologies1: ['Ruby on rails', 'css', 'Javascript'],
+    image: '/portfolio-Img/mobile-popup.png',
+    popTitle: 'Keeping track of hundreds of components',
+    popTechnologies2: ['Codekit', 'GitHub', 'JavScript', 'Bootstrap', 'Terminal', 'Codepen'],
+    btn1: 'See Live',
+    btn2: 'See Source',
+    linkToLiveSource: '#',
+    linkToSource: '#',
+  },
+  {
+    id: 6,
+    title: 'Multi-Post Stories Gain+Glory',
+    technologies: ['Ruby on rails', 'css', 'Javascript', 'html'],
+    popTechnologies1: ['Ruby on rails', 'css', 'Javascript'],
+    image: '/portfolio-Img/mobile-popup.png',
+    popTitle: 'Keeping track of hundreds of components',
+    popTechnologies2: ['Codekit', 'GitHub', 'JavScript', 'Bootstrap', 'Terminal', 'Codepen'],
+    btn1: 'See Live',
+    btn2: 'See Source',
+    linkToLiveSource: '#',
+    linkToSource: '#',
+  },
+];
+
+// all main cards goes there------------------------------------------------>
+
+const allSixCards = document.getElementById('all-six-cards');
+const popWindow1 = document.getElementById('popup-window');
+
+function display() {
+  let displayCard = '';
+  for (let i = 0; i < 6; i += 1) {
```

Comment:
- [ OPTIONAL ] Instead of using a static length in your loop, I will suggest that you use `cardData.lenght` so that whenever you add new data to the array, you don't need to change the value in the loop each time :wink:   
- eg. `  for (let i = 0; i < cardData.length; i += 1) {
Data goes here
}`